### ISSUE TITLE
ResNet50 performance

### ISSUE THREAD

**Issue by xvjiarui** [ajabri/videowalk#30]
Hi Allan,

Have you tried ResNet50 with your approach? I recently tried by myself, but get very low performance for the starting epoches, ~30 J&F mean. Any ideas?

**Comment by ajabri** [comment]
Hi @xvjiarui ,

I haven't experimented much with ResNet50, but using different networks (especially deeper networks) will require changing certain hyper-parameters; e.g. you may need a more extreme temperature (lower, like 0.01), and you may want to try a learning rate warmup.
